packages/app/src/__deprecated__/renderer/locales/default/en-US.json

```
@@ -383,6 +383,8 @@
   "module.filesManager.deletingFileModalTitle": "Deleting Files",
   "module.filesManager.deletingFileModalSubtitle": "Deleting in progress...",
   "module.filesManager.selectionNumber": "{num, plural, =-1 {All Files} one {# File} other {# Files}} selected",
+  "module.filesManager.uploadingModalTitle": "Uploading",
+  "module.filesManager.uploadingModalSubtitle": "Uploading files to device",
```

Maybe we can add the same copy as on the design? 
```suggestion
  "module.filesManager.uploadingModalSubtitle": "Files uploading...",
   "module.filesManager.uploadingModalDescription": "Please wait, this might take a few minutes",
```

---

`"module.filesManager.uploadingModalDescription"` is not showed anywhere, please add this text to modal too 😁

---

https://github.com/mudita/mudita-center/pull/914/commits/00e5926e3f3131f1f0b21bd6f401e739d57a90b7